presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8933,4 +8933,21 @@ public void testLateralJoin()
                 "SELECT * FROM (VALUES array[2, 2]) a(x) FULL OUTER JOIN LATERAL(VALUES x) ON true",
                 "line .*: LATERAL on other than the right side of CROSS JOIN is not supported");
     }
+
+    @Test
+    public void testCountAggregationOverScalar()
+    {
+        MaterializedResult actual = computeActual(
```

Comment:
why not to use plain `assertQuery`, then whole code of this method can be rewritten to:
```
assertQuery("SELECT COUNT(*) FROM (SELECT SUM(orderkey) FROM orders)");
```